extensions/ql-vscode/src/view/model-editor/ModelEditor.tsx

```diff
@@ -87,20 +89,21 @@ const ModelEvaluation = ({
   modifiedSignatures,
   onStartEvaluation,
   onStopEvaluation,
-  evaluationInProgress,
+  evaluationRun,
 }: {
   viewState: ModelEditorViewState;
   modeledMethods: Record<string, ModeledMethod[]>;
   modifiedSignatures: Set<string>;
   onStartEvaluation: () => void;
   onStopEvaluation: () => void;
-  evaluationInProgress: boolean;
+  evaluationRun: ModelEvaluationRunState | undefined;
 }) => {
   if (!viewState.showEvaluationUi) {
     return null;
   }
 
-  if (!evaluationInProgress) {
+  console.log("***** check evaluationRun", evaluationRun);
```

Comment:
I assume these `console.log`s shouldn't be merged in 😄

Reply:
Busted!